src/main/resources/application-localstack.yml

```diff
@@ -0,0 +1,14 @@
+hmpps.sqs:
+  reactiveApi: true
```

Comment:
not sure if you mean this to use the reactive api

Reply:
no definitely not!